specs/specCacheBooster.ts

```diff
@@ -8,7 +8,8 @@ import {CachePage} from '../pages';
 //
 // The timeouts in these tests are computed from that:
 // - 2 seconds to go from unknown state to non-cached state: 1 sec for polling + 1 sec of wiggle room
-// - 4 seconds to go from non-cached to cached state: 2 secs for computation + 1 sec for polling + 1 sec of wiggle room
+// - 10 seconds to go from non-cached to cached state: 8 secs for computation (usually about 2 but sometimes takes long time)
```

Comment:
The mission defines that the computation must take 2 seconds: https://github.com/fabric8-launcher/launcher-documentation/wiki/Mission:-Cache Obviously it can take a little more, but if it takes 8 seconds for you, then that sounds like a problem somewhere else.

Reply:
Usually it took something above 2 sec, but sometimes it got even 7 seconds. Possibly some catch after deployment.

Reply:
Weird. But the computation definitely doesn't take 8 seconds, so at least explain it by increasing wiggle room, not computation :-)

Reply:
OK I've changed the formulation

Reply:
Come on, please read the entire text :-)

Reply:
OK I did